interpretation/deepseismic_interpretation/azureml_pipelines/README.md

```diff
@@ -0,0 +1,174 @@
+
+## Running a Pipeline in AzureML
+Set the following environment variables:
+```
+BLOB_ACCOUNT_NAME
+BLOB_CONTAINER_NAME
+BLOB_ACCOUNT_KEY
+BLOB_SUB_ID
+AML_COMPUTE_CLUSTER_NAME
+AML_COMPUTE_CLUSTER_MIN_NODES
+AML_COMPUTE_CLUSTER_MAX_NODES
+AML_COMPUTE_CLUSTER_SKU
+```
+
+On Linux:
+`export VARIABLE=value`
+These can be set automatically in VSCode in an .env file, run with `source .env` in Linux or made into a `.bat` file to easily run from the command line in Windows. You can ask a team member for a .env file configured for our development environment to save time.
+
+Create a .azureml/config.json file in the project's root directory that looks like so:
+```json
+{
+"subscription_id": "<subscription id>",
+"resource_group": "<resource group>",
+"workspace_name": "<workspace name>"
+}
+
+```
+
+## Training Pipeline
+Here's an example of a possible config file:
+```json
+{
+    "step1":
+    {
+        "type": "PythonScriptStep",
+        "name": "process all files step",
+        "script": "process_all_files.py",
+        "input_datareference_path": "",
+        "input_datareference_name": "raw_input_data",
+        "input_dataset_name": "raw_input_data",
+        "source_directory": "src/first_preprocess/",
+        "arguments": ["--remote_run",
+        "--input_path", "input/",
+        "--output_path", "normalized_data"],
+        "requirements": "src/first_preprocess/preprocess_requirements.txt",
+        "node_count": 1,
+        "processes_per_node": 1
+    },
+    "step2":
+    {
+        "type": "PythonScriptStep",
+        "name": "prepare files step",
+        "script": "prepare_files.py",
+        "input_datareference_path": "normalized_data/",
+        "input_datareference_name": "normalized_data_conditioned",
+        "input_dataset_name": "normalizeddataconditioned",
+        "source_directory": "src/second_preprocess",
+        "arguments": ["split_train_val",
+        "patch",
+        "--label_file", "label.npy",
+        "--output_dir", "splits/",
+        "--stride=25",
+        "--patch=100.",
+        "--log_config", "configs/logging.conf"],
+        "requirements": "src/second_preprocess/prepare_files_requirements.txt",
+        "node_count": 1,
+        "processes_per_node": 1,
+        "base_image": "pytorch/pytorch"
+    },
+    "step3":
+    {
+        "type": "MpiStep",
+        "name": "train step",
+        "script": "train.py",
+        "input_datareference_path": "normalized_data/",
+        "input_datareference_name": "normalized_data_conditioned",
+        "input_dataset_name": "normalizeddataconditioned",
+        "source_directory": "train/",
+        "arguments": ["--splits", "splits",
+        "--train_data_paths", "normalized_data/file.npy",
+        "--label_paths", "label.npy"],
+        "requirements": "train/requirements.txt",
+        "node_count": 1,
+        "processes_per_node": 1,
+        "base_image": "pytorch/pytorch"
+    }
+}
+```
+  
+If you want to create a train pipeline:
```

Comment:
You could add some text around the benefits of creating an AML pipeline, like why are we doing this in the first place?

Reply:
I've added some summary high level on AML at the top of the README.md in this commit https://github.com/microsoft/seismic-deeplearning/pull/195/commits/5aacf791335c4982b6c35f374a7ebaa0748be563. Do you find this to be an appropriate level of documentation here?

Reply:
looks good